File: app/src/main/java/za/co/woolworths/financial/services/android/ui/activities/product/dynamicyield/response/request/Properties.java
package za.co.woolworths.financial.services.android.ui.activities.product.dynamicyield.response.request;

import java.io.Serializable;

public class Properties implements Serializable {
    private String keywords;

    private String dyType;

    public Properties(String keywords, String dyType) {
        this.keywords = keywords;
        this.dyType = dyType;
    }

    public String getKeywords() {
        return this.keywords;
    }

    public void setKeywords(String keywords) {
        this.keywords = keywords;
    }

    public String getDyType() {
        return this.dyType;
    }

    public void setDyType(String dyType) {
        this.dyType = dyType;
    }
}

File: app/src/main/java/za/co/woolworths/financial/services/android/ui/fragments/product/detail/DyChangeAttribute/Request/Properties.kt
package za.co.woolworths.financial.services.android.ui.fragments.product.detail.DyChangeAttribute.Request

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Properties(
    val attributeType: String? = null,
    val attributeValue: String? = null,
    val dyType: String? = null,
    val keywords: String? = null,
    val value: String? = null,
    val currency: String? = null,
    val quantity: Int? = null,
    val productId: String? = null,
    val color: String? = null,
    val sku_id: String? = null,
    val hashedEmail: String? = null,
    val size: String? = null,
    val sortBy: String? = null,
    val sortOrder: String? = null,
    val filterType: String? = null,
    val filterStringValue: String? = null,
    val uniqueTransactionId: String? = null,
    val cart: List<Cart>? = null
): Parcelable